test/index.js

```diff
@@ -741,6 +741,42 @@ it('accepts request object in validateFunc', (done) => {
     });
 });
 
+it('includes additional attributes in WWW-Authenticate header', (done) => {
+    const server = new Hapi.Server();
```

Comment:
I was hoping lab defaulted to that. Doesn't look like it's catching the missing new line for the arrow function but correctly spots it for traditional function declaration. I'll open an issue with [hapi-scope-start](https://github.com/continuationlabs/hapi-scope-start)